deployment/scripts/install.sh

```diff
@@ -277,11 +277,11 @@ fi
 echo "(Re-)Creating deployed objects"
 if [ "${KEEP_SUPPORT}" != true ]; then
 	oc process logging-support-template | oc delete -f - || :
-	oc delete serviceaccount,service,route --selector logging-infra=support
+	oc delete serviceaccount,service,route,oauthclient --selector logging-infra=support
```

Comment:
Can't do this, I know it probably seems paranoid but the role allows the deployer to create and delete oauthclient... not get/list (so that it can't be used to retrieve oauth client secrets). In order to delete by label it has to be able to list, so this fails. That's why the delete above is necessary (it probably deserves a comment though).